test.js

```diff
@@ -276,6 +276,28 @@ test('rehypeExternalLinks', async (t) => {
     'should add dynamic `target`, `rel` to links'
   )
 
+  t.equal(
+    String(
+      await rehype()
+        .use({settings: {fragment: true}})
+        .use(rehypeExternalLinks, {test: (node) => node.properties.href === 'http://example.com'})
+        .process('<a href="http://example.com">http</a>')
+    ),
+    '<a href="http://example.com" rel="nofollow">http</a>',
+    'should add rel to a link that matches the test function'
+  )
+
+  t.equal(
+    String(
+      await rehype()
+        .use({settings: {fragment: true}})
+        .use(rehypeExternalLinks, {test: (node) => node.properties.href === 'http://foobar.com'})
```

Comment:
Please change these checks from `node.properties.href` -> `node.properties && node.properties.href`. See CI.

And then run `npm test` to format everything!

Reply:
`npm test` fails due to missing coverage (as does the main branch for me), but I've formatted it with `npm format` 👍